### PR TITLE
adding test and the missing container to docker build

### PR DESCRIPTION
**Pull request by bam241**
adding test for docker img change as done for dagmc

### CODE REVIEW

**Comment by bam241** [comment]
status can be found : [here](https://github.com/bam241/pyne/actions/workflows/docker_publish.yml)

**Comment by gonuke** [comment]
It looks like pymoab is missing from some of these images??

**Comment by bam241** [comment]
yes was using build_pymoab instead of enable_pymoab...

**Comment by gonuke** [comment]
I think this is now failing on our flaky `test_variancereduction.py` issue

**Comment by bam241** [comment]
yes !
we might need moab 5.3 ?

I am also making some test using cache to win some time of building time

**Comment by gonuke** [comment]
I didn't think PyNE failed because of MOAB though???

File: .github/workflows/docker_publish.yml [hunk 1]
```diff
@@ -37,78 +35,269 @@ jobs:
...
-            install_openmc=YES
-          tags: ghcr.io/pyne/ubuntu_18.04_py3_dagmc_pymoab_openmc_pyne-deps:latest
+            enable_pymoab=${{ env.PYMOAB }}
+            build_dagmc=${{ env.DAGMC }}
+            build_hdf5=${{ env.HDF5 }}
+            install_openmc=${{ env.HDF5 }}
+          cache-from: type=local,src=/tmp/.buildx-cache-${{ github.sha }}
+          tags: ghcr.io/${{ github.repository_owner }}/ubuntu_18.04_py3${{ matrix.hdf5 }}${{ matrix.dagmc }}${{ matrix.pymoab }}${{ matrix.openmc }}_pyne-deps:latest
+
+  cleanup_cash:
```
Comment (gonuke) [inline .github/workflows/docker_publish.yml]:
```suggestion
  cleanup_cache:
```

**Comment by bam241** [comment]
Everything worked as I wanted.

added the guard to retag on PyNE only.

this looks good on my side @gonuke

**Comment by bam241** [comment]
see : https://github.com/bam241/pyne/actions/runs/1046794230

**Comment by gonuke** [comment]
I'll wait for this: https://github.com/bam241/pyne/actions/runs/1046870634

**Comment by gonuke** [comment]
This looks like it's working well enough to merge - thanks @bam241

**Comment by gonuke** [comment]
Oops - I ignored the missing CHANGELOG entry!

**Pull request merged**